src/parser/jobs/ast/modules/Draw.tsx

```diff
@@ -256,28 +267,28 @@ export default class Draw extends Module {
 			SUGGESTION: Didn't use sleeve draw enough
 		*/
 		const sleevesMissed = Math.floor(this.sleeveTotalDrift / this.data.actions.SLEEVE_DRAW.cooldown)
-		if (this.sleeveUses > 0 && sleevesMissed > 0) {
+		if (this.sleevesUsed > 0 && sleevesMissed > 0) {
 			this.suggestions.add(new TieredSuggestion({
 				icon: this.data.actions.SLEEVE_DRAW.icon,
 				content: <Trans id="ast.draw.suggestions.sleeve-uses.content">
-						Use <ActionLink {...this.data.actions.SLEEVE_DRAW} /> more frequently. It should be paired with every other <ActionLink {...this.data.actions.DIVINATION} /> to stack buffs at the same time. <ActionLink {...this.data.actions.LIGHTSPEED} /> can be used to weave card abilities.
+						Use <DataLink action="SLEEVE_DRAW" /> more frequently. It should be paired with every other <DataLink action="DIVINATION" /> to stack buffs at the same time. <DataLink action="LIGHTSPEED" /> can be used to weave card abilities.
```

Comment:
We generally use soft language, so rather than "Use", a more passive phrasing would be "Try to use".

Reply:
updated